docs/guide/topics/error_handling.rst

```diff
@@ -0,0 +1,361 @@
+    :emphasize-lines: 8
+
+    class ExampleCog(commands.Cog):
+
+        async def cog_check(ctx):
+            """ Custom check for this cog only. """
+            if ctx.guild and ctx.guild.id != 336642139381301249:
+                raise commands.DisabledCommand
+
+        async def cog_command_error(self, ctx, error):
+            """ This error handler is only for this specific Cog. """
+
+            if isinstance(error, commands.DisabledCommand):
+                await ctx.send(f"The command {ctx.command!r} cannot be used in this server!")
+      
+In a :class:`~.ext.commands.Command`
+
+************************************
+
+
+To handle errors for a specific command, we decorate the function with :meth:`~ext.commands.Command.error`, which will 
+only get called for that specific command.
+
+You can use the same function for multiple commands too, simply by adding multiple decorators on top of it.
+
+**Example**
+
+.. code-block:: python3
+    :emphasize-lines: 17, 18
+
+    @bot.command()
+    @commands.has_permissions(kick_members=True)
+    @commands.bot_has_permissions(kick_members=True)
+    async def kick(ctx, member: discord.Member):
+        """ Kicks a member from this server. """
+        await member.kick()
+        await ctx.send(f"Kicked {member}! Bye bye.")
+
+    @bot.command()
+    @commands.has_permissions(ban_members=True)
+    @commands.bot_has_permissions(ban_members=True)
+    async def ban(ctx, member: discord.User):
+        """ Bans a member from this server. """
+        await ctx.guild.ban(user)
+        await ctx.send(f"Banned {member}! Begone.")
+
+    @kick.error
+    @ban.error
+    async def kick_error(ctx, error):
+        """ This error handler is only for the `kick` and `ban` commands """
+
+        if isinstance(error, commands.MissingPermissions):
+            formatted_perms = ', '.join(error.missing_perms)
+            await ctx.send(f'You are missing the following permissions: {perms}')
+
+        elif isinstance(error, commands.BotMissingPermissions):
+            formatted_perms = ', '.join(error.missing_perms)
+            await ctx.send(f'I am missing the following permissions: {perms}')
+
+
+.. warning::
+    After the local error handlers get called, the global error handler gets called too. If you're handling unhandled errors 
+    in it, do not handle them in local error handlers, as this may cause them to be handled twice or more.
+
+Using Global and Local Error Handlers Together
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Some times, you may want or need to use local and global error handlers, in these cases, there are multiple ways of checking if 
+the error has already been handled before. There are many approaches on how to do this:
+
+- Using the :meth:`Cog.has_error_handler <discord.ext.commands.Cog.has_error_handler>`
+  and :meth:`Command.has_error_handler <discord.ext.commands.Command.has_error_handler>`
+  methods in your global error handler.
+
+**Example**
```

Comment:
I don't think an example is needed here. You can simply explain this- It's a single conditional.